docker-compose.development.yml

```diff
@@ -4,6 +4,13 @@ services:
     build:
       context: ./api
       dockerfile: Dockerfile.dev
+    entrypoint:
```

Comment:
Thanks for this.

Rather than make these changes here, could we update `Dockerfile.dev`?

If I'm understanding correctly, we could get away with replacing

```
ENTRYPOINT [ "python" ]

CMD [ "app.py" ]
```
with

```
CMD [ "flask", "run", "--host=0.0.0.0", "--port=5001"]
```

yes?

Reply:
Agreed.